src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -26,6 +26,7 @@ export default () => `
 						<li><a href="command:notebook.command.new">${escape(localize('welcomePage.newNotebook', "New notebook"))}</a></li>
 						<li class="mac-only"><a href="command:workbench.action.files.openLocalFileFolder">${escape(localize('welcomePage.openFileMac', "Open file"))}</a></li>
 						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFile">${escape(localize('welcomePage.openFileLinuxPC', "Open file"))}</a></li>
+						<li><a href="command:azdata.resource.deploy">${escape(localize('welcomePage.deploySQL', "Deploy SQL Server…"))}</a></li>
```

Comment:
How does this work in Insiders vs. Stable release? Given we'll be shipping only in Insiders for a while, I am concerned that this option will appear in Stable but not be usable. How will we resolve that?

Reply:
@kevcunnane , addressed this issue in my latest commit.